### DISCUSSION TITLE
Weird behaviour with Xdrip plus with Nightscout and the Glooko Bridge

### DISCUSSION THREAD

**Discussion by Luca-7JGKP** [NightscoutFoundation/xDrip#3214] [Bug reports] [answered]
Hello, I use the Ypsomed pump with the CamAPSFX Loop. To get the data for XDrip+ I use the data source Companion App.
The Phone is a Samsung A22 and the Xdrip Version is the Nightly build 21st Nov 2023
Let's just get to the problem: That's the prediction. Xdrip + Automatically doubles the values ​​he gets from Nightscout (Nightscout Import from Glooko). (Carbohydrates and insulin) that is the first problem.
Prediction in Nightscout:
![Screenshot_20231128_143453_Firefox](https://github.com/NightscoutFoundation/xDrip/assets/32716711/4c8ecfeb-4957-4c4b-a627-463a0b12262e)
Prediction in Xdrip+:
![Screenshot_20231128_143441_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/32716711/1be55674-1129-4cf6-bc2d-c8ee9ff13bd7)

The second problem is that Xdrip+ takes notes every time. So-called time stamps that make no sense.
![Screenshot_20231128_145613_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/32716711/cbb09fca-58f1-4978-ae42-d3844563f1ec)

Since it's only the values ​​in Xdrip that are so strange and not in nightscout, I believe it's a bug in Xdrip +.

Thanks

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
On your screenshots I see Nightscout 4U 24g, same on xDrip+. Do you mean the original treatment was 2U 12g? The events are from your pump. They probably are in Nightscout but you don't see the flags or they are visualized differently. xDrip+ doesn't know how to represent them so it shows notes. I would be curious to know how Nightscout sees them.  Can you show the pump pill in Nightscout?

**Comment by Luca-7JGKP** [comment]
@psonnera don't see a pump pill in Nightscout because this module is not activated at all. How i do that?
The only thing that surprises me is Xdrip's calculation. It displays the values ​​correctly. I just found out.

Edit:
The Marker are the Temporary Basal Rate
![Screenshot 2023-11-28 at 18-43-14 182 -1 →](https://github.com/NightscoutFoundation/xDrip/assets/32716711/ee396fdc-e3dd-4670-9f34-33380a5f40b9)

Temporary basal rate see screenshot. How do I get Xdrip+ not to import it? Or Nightscout not imported.

**Comment by psonnera** [comment]
I rhink the issue is on Nightscout side: the basal change is not handled correctly by Nightscout. Since the plugin is still in development that might be the reason.

**Comment by Luca-7JGKP** [comment, marked as answer]
We found a Workaround.
The Comments are the Fault because the Markers.
So we say that the Plugin dont Import the Comments now are all fine!
